index.html

```diff
@@ -4,11 +4,12 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Quiz-App</title>
-    <link rel="stylesheet" href="styles.css" />
+    <link rel="stylesheet" href="scss/styles.scss" />
     <link
       href="https://fonts.googleapis.com/icon?family=Material+Icons"
```

Comment:
Ne Icon-Font für bookmarks, die wir in CSS bauen können, lohnt sich nicht, finde ich.